src/webserve_kq/Server.cpp

```diff
@@ -0,0 +1,78 @@
+//
+// Server
+// Server class stores server information and set listening socket(s) belonging to this server
+//
+
+#include "../../include/Server.hpp"
+#include "../../include/macro.hpp"
+
+
+Server::Server(std::vector<int> ports, std::string name, std::string root_dir, int kq):
```

Comment:
is it 1 port or multiple ports? is name the serverName? rootDir.

Reply:
One server might have multiple listening ports. And several servers can use the same port as a listening port.
These things still make me feel confused...